src/cloudai/__main__.py

```diff
@@ -96,29 +96,29 @@ def parse_arguments() -> argparse.Namespace:
         ),
     )
     parser.add_argument(
-        "--system_config_path",
+        "--system-config",
```

Comment:
I like these updates. "_path" is not essential. Having "-dir" at the end allows users to know whether they should provide a file or directory.

cc: @srinivas212